apps/heureka-next/src/components/Services/ServicesList/ServicesList.test.tsx

```diff
@@ -0,0 +1,25 @@
+/*
+ * SPDX-FileCopyrightText: 2024 SAP SE or an SAP affiliate company and Juno contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import React from "react"
+import { render, screen } from "@testing-library/react"
+import { ServicesList } from "./ServicesList"
+import { TestProvider } from "../../../mocks/TestProvider"
```

Comment:
I find interesting the thing with the `TestProvider`. May be you can show me it any time...